interfaces/utils/path_patterns.go

```diff
@@ -138,7 +138,8 @@ func createRegex(pattern string, glob GlobFlags) (string, error) {
 				itemCountInGroup[currentGroupLevel]++
 				regex += "|"
 			} else {
-				return "", fmt.Errorf("cannot use ',' outside of group or character class")
+				// treat commas outside of groups as literal commas
+				regex += ","
```

Comment:
It might be desirable to treat commas outside of groups/classes as a class containing only `,`; that is, `regex += "[,]"`, since that is equivalent and removes all ambiguity.

Also, could use `regexp.QuoteMeta(",")` or `regexp.QuoteMeta("[,]")` just to be safe.